samples/java-spring-eventsourced-customer-registry-subscriber/docker-compose-integration.yml

```diff
@@ -0,0 +1,42 @@
+# this docker-compose is used by the integration tests. To run it, we must first publish
+# java-spring-customer-registry image locally. To do so, run the following command from the root of the project:
+# mvn clean install docker:build
+
+version: "3"
+services:
+  kalix-proxy:
+    image: gcr.io/kalix-public/kalix-proxy:1.1.9-M2
+    depends_on:
+      - kalix-proxy-customer-registry
+    ports:
+      - "9001:9000"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      JAVA_TOOL_OPTIONS: >
+        -Dconfig.resource=dev-mode.conf
+        -Dlogback.configurationFile=logback-dev-mode.xml
+        -Dkalix.dev-mode.service-port-mappings.customer-registry=host.docker.internal:9000
```

Comment:
Proxy 1.1.9-M2 can use this property to do local resolution when running in dev-mode. 

When I added it, I was in doubt if we should accept settings with only the port. Like in, `.customer-registry=9000` as opposed to `.customer-registry=host.docker.internal:9000`

It will be much simpler if we said: `customer-registry=9000`. Then the proxy would add the `host.docker.internal` and the user function would add `localhost`. 

But I'm not sure if other docker tools (eg: colima, podman) will also have `host.docker.internal` as a default. Docker Desktop does it and that's also how it behaves in Linux. I can imagine that all other tools will simply follow it, but not sure.

Reply:
I think its wise to accept both host and port. Colima didn't support `host.docker.internal` until very recently I believe.

Reply:
Yes, indeed. 

Eventually, we could also say. If the host is omitted, it defaults to `host.docker.internal` otherwise to whatever the user declares. We let it like that for now and be less strict later.